File: GTTS/INL.UserService.Client/Models/AppUser_Item.cs
namespace INL.UserService.Client.Models
{
    public class AppUser_Item : IAppUser_Item
    {
		public long AppUserID { get; set; }
		public string First { get; set; }
		public string Last { get; set; }
		public string Middle { get; set; }
		public string FullName { get; set; }
		public string PositionTitle { get; set; }
		public string EmailAddress { get; set; }
		public string PhoneNumber { get; set; }
		public string PicturePath { get; set; }
		public int? CountryID { get; set; }
		public string CountryName { get; set; }
		public int? PostID { get; set; }
		public string PostName { get; set; }
		public int? DefaultBusinessUnitID { get; set; }
		public string DefaultBusinessUnitName { get; set; }
		public string DefaultBusinessUnitAcronym { get; set; }
	}
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.UserService.Models/app-user_item.ts


import { IAppUser_Item } from './iapp-user_item';

export class AppUser_Item implements IAppUser_Item {
  
	public AppUserID: number = 0;
	public First: string = "";
	public Last: string = "";
	public Middle: string = "";
	public FullName: string = "";
	public PositionTitle: string = "";
	public EmailAddress: string = "";
	public PhoneNumber: string = "";
	public PicturePath: string = "";
	public CountryID?: number;
	public CountryName: string = "";
	public PostID?: number;
	public PostName: string = "";
  
}


